blocks.js

```diff
@@ -7553,13 +7556,43 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-            // } else if (key.indexOf('§_def') === 0) {
-            //     menu.addItem(choices[key].blockInstance(), choices[key]);
+         // } else if (key.indexOf('§_def') === 0) { 
+         //     menu.addItem(choices[key].blockInstance(), choices[key]);
+            } else if (key.charAt(key.length - 1) == '{') {
+                // Submenu
+                // 0x25ba = Unicode "BLACK RIGHT-POINTING POINTER"
+                // (not to be confused with "black right-pointing TRIANGLE")
+                menu.addItem(key.substring(0,key.length - 1).
+							 concat(String.fromCharCode(0x25ba)),
```

Comment:
Also can we _please_ add some space between the last character and the pointer. Any amount will do, but it looks that we're likely to see menus that look like this, and I'm just going to be driven mad when I notice it. (and users shouldn't have to worry about proper spacing.
```js
> 'a' + '\u25ba'
'a►'
```